src/Triangle.java

```diff
@@ -0,0 +1,9 @@
+public class Triangle {
+    double area;
+    int height;
+    int length;
+
+    void setArea() {
+        area = (height * length) / 2;
```

Comment:
Question 1: what is mathematical order of operations?
Question 2: is this working properly? It does not look like it would.

Reply:
PEMDAS( parenthesis, expo, mutliply, divide, add, sub
![tri](https://user-images.githubusercontent.com/77342344/165525724-7eff6a01-b21d-4fda-a1c5-4a11818142f1.PNG)
)
yep it returned as it should

Reply:
nope. 
Try:
		triange.height = 3;
		triange.length = 1;
		triange.setArea();

What's the result vs. what should be the result?